testing/a11y/src/Component.cy.tsx

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import * as allExamples from 'examples';
+import { ThemeProvider } from '@itwin/itwinui-react';
+
+describe('Should have no WCAG violations', () => {
+  Object.entries(allExamples).forEach(([name, Component]) => {
+    it(name, () => {
+      cy.mount(
+        <ThemeProvider theme='dark' style={{ height: '100vh' }}>
+          <Component />
+        </ThemeProvider>,
+      );
+      cy.injectAxe({
+        axeCorePath: Cypress.env('axeCorePath'),
+      });
+      cy.checkA11y(undefined, undefined, (violations) => {
+        const violationData = violations.map(
+          ({ id, impact, description, nodes }) => ({
+            id,
+            impact,
+            description,
+            nodes: nodes.length,
+          }),
+        );
```

Comment:
Lets make the table more concise and useful.

```suggestion
        const violationData = violations.map(({ id, help }) => ({
          Component: name,
          'Rule ID': id,
          Description: help,
        }));
```

![](https://github.com/iTwin/iTwinUI/assets/9084735/25662a86-291d-4931-b952-82deaf807f39)

Reply:
done!